app/uk/gov/hmrc/partnershipidentificationfrontend/models/PartnershipInformation.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.partnershipidentificationfrontend.models
+
+import play.api.libs.functional.syntax.{unlift, _}
+import play.api.libs.json._
+
+
+case class PartnershipInformation(sautr: Option[String],
+                                  postcode: String)
+
+object PartnershipInformation {
```

Comment:
If the retrievePartnershipInformation method is not needed, do we need this model?